src/assets/PugAsset.js

```diff
@@ -0,0 +1,121 @@
+const Asset = require('../Asset');
+const HTMLAsset = require('./HTMLAsset');
+
+const load = require('pug-load');
```

Comment:
Probably best to use localRequire('..') for the pug dependencies, instead of shipping them with parcel

Reply:
Need to replace `pug-*` packages with `localRequire('...', this.name)`?

Reply:
Fixed